subprojects/docs/src/samples/buildCache/developer-ci-setup/readme.xml

```diff
@@ -17,6 +17,6 @@
 <sample>
     <para>
         Recommended cache configuration: Developer push to a local build cache and pull from local and remote build cache,
-        continuous integration server pushes to and pulls from the remote cache.
+        continuous integration server pushes to and pulls from both the local and remote caches.
```

Comment:
I guess that should be `stores` and `loads` instead of `pushes` and `pulls`, since we got rid of that naming some time ago.

Reply:
Fixed, please take another look.